module/zfs/dbuf.c

```diff
@@ -400,7 +403,7 @@ dbuf_hash_insert(dmu_buf_impl_t *db)
 	uint32_t i;
 
 	blkid = db->db_blkid;
-	hv = dbuf_hash(os, obj, level, blkid);
+	hv = db->db_hv;
 	idx = hv & h->hash_table_mask;
```

Comment:
looks like you could inline `hv` into this one use:
```c
idx = dv->db_hv & h->hash_table_mask;
```

Reply:
Done.